libraries/dagster-hex/dagster_hex/asset_decorator.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Callable, Optional
+
+
+import dagster as dg
+from dagster._annotations import experimental
+
+
+"""
+
+HexWorkspace
+
+Figure out _inputs_
+
+"""
+
+
+class HexWorkspace(dg.ConfigurableResource):
+    api_key: str
+
+
+@experimental
+def hex_project_asset(
+    *,
+    project_id: str,
+    workspace: HexWorkspace,
+    name: Optional[str] = None,
+    group_name: Optional[str] = None,
+) -> Callable[[Callable[..., Any]], dg.AssetsDefinition]:
```

Comment:
Is this meant to be in the PR? It's not used in the example.

Also, should we use a HexWorkspace and a decorator for Hex? I assume that using HexResource makes more sense at the moment since it is implemented in the PR?

Reply:
Ah yes, I was considering creating a `@hex_assets` multi-asset with a workspace similar to other integrations, but I am going to remove for now. Simply having a resource for the time being should be sufficient for most user's needs. Thanks for pointing that out.